sdk/appconfiguration/azure-spring-cloud-appconfiguration-config-web/src/test/java/com/azure/spring/cloud/config/web/pushbusrefresh/AppConfigurationBusRefreshEndpointTest.java

```diff
@@ -19,6 +19,7 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import org.junit.After;
```

Comment:
So we're using the jupiter engine to run junit4 tests?

Reply:
Instead of using `org.junit.jupiter.api.Test`.

Reply:
Should be updated now. I was mainly focusing on the non-web library changes.